openhands/cli/settings.py

```diff
@@ -304,10 +306,8 @@ async def modify_llm_settings_advanced(
     ):
         return  # Return on exception
 
-    # TODO: check for empty string inputs?
-    # Handle case where a prompt might return None unexpectedly
-    if custom_model is None or base_url is None or api_key is None or agent is None:
-        return
```

Comment:
Same here probably still want this statement

Reply:
Similarly, this is unreachable code.